newIDE/app/src/ProjectManager/index.js

```diff
@@ -130,6 +132,7 @@ type ItemProps = {|
   onMoveUp: () => void,
   canMoveDown: boolean,
   onMoveDown: () => void,
+  addonMenuOptions?: Array<Object>,
```

Comment:
Let's call this "extraMenuItems" and use a proper typing instead of Object.

Reply:
Done.